### PR TITLE
A2 11

### PR DESCRIPTION
**Pull request by Sedgy**
Feel free to comment changes on how the article content is to be displayed, currently just p tags with breaks and altered strong tag for the emphasis section (increase font size). 

Also first pull request ¯\\_(ツ)_/¯ be gentle.

### CODE REVIEW

File: static/src/assets/scss/pages/_article.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	justify-content: center;
+
+	h1, p{
+		color: $white;
+		text-shadow: 1px 1px 2px $black;
+	}
+
+}
+
+.article-content{
```
Comment (lrobert142) [inline static/src/assets/scss/pages/_article.scss]: Could we add some padding for the image to offset it from the text?

File: static/src/pages/article.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        
+    </div>
+    <div class="row">
+        <div class="medium-4 column" data-equalizer-watch><img src="https://placehold.it/350x350"></div>
+        <div class="medium-8 column">
+            <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu.
+                <br>
+                <br>
+                <strong>In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus.</strong>
```
Comment (lrobert142) [inline static/src/pages/article.html]: I feel that we should have this as 3 paragraphs, not just separated by `<br>`s

File: static/src/pages/article.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+<section class="latest-news section box" data-equalizer data-equalize-on="medium">
+    <div class="row">
+        <h2>Related News</h2>
+        <div class="medium-4 columns card" data-equalizer-watch>
+            <div class="card-section">
+                <span class="date">11 April 2017</span>
+                <h3>You Wont Believe this thing!</h3>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu.</p>
+                <a href="#">Read more ></a>
```
Comment (lrobert142) [inline static/src/pages/article.html]: Don't need the ">" after read more

File: static/src/assets/scss/pages/_article.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	background-size: cover;
+	height: $hero-height;
+	text-align: center;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	h1, p{
+		color: $white;
+		text-shadow: 1px 1px 2px $black;
```
Comment (lrobert142) [inline static/src/assets/scss/pages/_article.scss]: If possible, move this to be a variable in _custom-settings.scss (so we can use it across all text shadow headings)

File: static/src/assets/scss/pages/_article.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+	}
+
+}
+
+.article-content{
+	strong{
+		font-size: 20px;
```
Comment (lrobert142) [inline static/src/assets/scss/pages/_article.scss]: This font size is a bit too large. Could we use $font-size-content instead?

File: static/src/pages/article.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        </div>
+    </div>
+</section>
+<section class="article-content" data-equalizer data-equalize-on="medium">
+    <div class="row">
+            <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu.</p>
+        
+    </div>
+    <div class="row">
+        <div class="medium-4 column" data-equalizer-watch><img src="https://placehold.it/350x350"></div>
```
Comment (lrobert142) [inline static/src/pages/article.html]: Image ends up a bit squashed on medium-sized (tablet-sized) screens. Could we maybe put the image to the top of the article when we reach this size?

File: static/src/assets/scss/pages/_article.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
+.article-hero {
+    background: url('https://images.pexels.com/photos/208826/pexels-photo-208826.jpeg?w=940&h=650&auto=compress&cs=tinysrgb') 50% no-repeat;
```
Comment (axrs) [inline static/src/assets/scss/pages/_article.scss]: Grab an image from a placeholder site (like [http://www.placehold.it/](http://www.placehold.it/))

We can override the background with an inline style when the article is actually integrated into WP.

File: static/src/pages/article.html [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    <div class="row">
+        <div class="medium-8 medium-centered columns">
+            <h1>Firm Takes Skills Rein</h1>
+            <p>by Mr Author Dude</p>
+        </div>
+    </div>
+</section>
+<section class="article-content" data-equalizer data-equalize-on="medium">
+    <div class="row">
+        <div class="large-4 column" data-equalizer-watch><img src="https://placehold.it/350x350" class="thumbnail"></div>
```
Comment (axrs) [inline static/src/pages/article.html]: Shouldn't need data-equalizer here.

File: static/src/pages/article.html [hunk 9]
```diff
@@ -0,0 +1,55 @@
+<section class="article-hero">
+    <div class="row">
+        <div class="medium-8 medium-centered columns">
+            <h1>Firm Takes Skills Rein</h1>
+            <p>by Mr Author Dude</p>
+        </div>
+    </div>
+</section>
+<section class="article-content" data-equalizer data-equalize-on="medium">
```
Comment (axrs) [inline static/src/pages/article.html]: Does the article need equalizing?

File: static/src/pages/article.html [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+    </div>
+    <div class="row">
+        <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu.</p>
+    </div>
+    <div class="row">
+        <div class="column row">
```
Comment (axrs) [inline static/src/pages/article.html]: No need for class column row.

**Pull request merged**